yapapi/strategy/wrapping_strategy.py

```diff
@@ -0,0 +1,29 @@
+from yapapi.props.builder import DemandDecorator, DemandBuilder
+
+from .base import MarketStrategy
+
+
+class WrappingMarketStrategy(DemandDecorator):
+    """
+    Helper class which allows other/user defined strategies to wrap some other strategies,
+    without overriding the attributes (e.g. defaults) defined on the derived-from strategy.
+
+    WrappingMarketStrategy classes are unusable on their own and always have to wrap some base
+    strategy.
+    """
+
+    base_strategy: MarketStrategy
```

Comment:
Also here, I think we can wrap a wrapper.

Reply:
yes